Prolangle/Pages/Home.razor

```diff
@@ -69,31 +76,104 @@ else
 
 	private List<ILanguage> AvailableLanguages => LanguagesProvider!.Languages.Where(l => !RevealedLanguages.Contains(l)).ToList();
 	private List<ILanguage> RevealedLanguages { get; } = [];
-	private ILanguage TargetLanguage => GuessGame!.MetadatumGameLanguage;
+	private ILanguage? TargetLanguage { get; set; }
+	private Game? Game;
 
 	// revisit these fields once #22 is resolved
 	private int guessCount;
 	private bool won;
 
-	private void OnLanguageSelected(ILanguage language)
+	protected override async Task OnInitializedAsync()
 	{
+		await DbManager.OpenDb();
+
+		DbManager.ActionCompleted += (_, args) => Logger.LogInformation("IndexedDB: {Message} => {Outcome}", args.Message, args.Outcome);
+	}
+
+	protected override async Task OnParametersSetAsync()
+	{
+		var games = await DbManager.GetRecords<Game>(Game.StoreName);
+		var game = games.FirstOrDefault(g => g.Seed == GuessGame.Seed && g.TypeId == GameType.PropertiesGameTypeId);
```

Comment:
doesn't this need to filter by date? or should the user be able to continue past the day?

Reply:
No, the `Seed` should suffice. After all, the seed determines the target language, not the day. The day is only used to "seed the seed". In development, for example, it is dependent on the current hour.